Rakefile

```diff
@@ -100,33 +100,8 @@ namespace :demo do
   end
 end
 
-desc 'Run Carthage update'
```

Comment:
If at some point Carthage support is needed for this SDK, these tasks can be pulled from `braintree_ios`.